services/jobs/src/main/java/com/dremio/service/jobs/QueryProfileParser.java

```diff
@@ -78,6 +78,7 @@
   public QueryProfileParser(final JobId jobId, final QueryProfile queryProfile) throws IOException {
     mapper = new ObjectMapper();
     mapper.configure(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES, true);
+    mapper.configure(JsonParser.Feature.ALLOW_NON_NUMERIC_NUMBERS,true);//this is added for solved this issue. https://community.dremio.com/t/non-standard-token-nan/637
```

Comment:
Can you change the comment to:
```
Allow parsing NaN in query profiles. See https://community.dremio.com/t/non-standard-token-nan/637

```
and put it just above?

Reply:
ok